projects/slipshow/programs/slipshow/tests/shell.nix

```diff
@@ -0,0 +1,41 @@
+{
+  sources,
+  ...
+}:
+{
+  name = "slipshow presentation";
+
+  nodes = {
+    machine =
+      { pkgs, ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.programs.slipshow
+          sources.examples.slipshow.basic
+        ];
+
+        environment.etc."slipshow".source = pkgs.fetchFromGitHub {
+          owner = "meithecatte";
+          repo = "bbslides";
+          rev = "ce1c08cafa71ae36dda8cc581956548b8386ae16";
+          hash = "sha256-sOydmvtDeMhNejDkwlsXdrbwtqN6lcNnzTnGzBVRFxA=";
+        };
+
+      };
+  };
+
+  testScript =
+    { nodes, ... }:
+    ''
+      start_all()
+
+      # it may take around a minute to compile the file and serve it
+      machine.succeed("slipshow serve /etc/slipshow/bbslides.md &>/dev/null &")
+
+      # slipshow serves defaultly on :8080 and unfortunately cannot
+      # be changed currently
```

Comment:
Thanks a lot @ethancedwards8 for packaging slipshow!

Just wanted to mention that since the recently released v0.7.0, it is possible to specify the port with `-p` or `--port`.